cmd/ci-operator-config-mirror/main.go

```diff
@@ -44,8 +44,10 @@ func gatherOptions() options {
 	fs.BoolVar(&o.clean, "clean", true, "If the `to-org` folder already exists, then delete all subdirectories")
 
 	o.WhitelistOptions.Bind(fs)
-	fs.Parse(os.Args[1:])
-	return o
+	if err := fs.Parse(os.Args[1:]); err != nil {
+		return o, fmt.Errorf("failed to parse flags: %w", err)
+	}
+	return o, nil
```

Comment:
I think these are the one we can actually easily ignore because of the `flag.ExitOnError` -- this would make even the callsites simpler.

Reply:
But I do not insist: if you feel like you already spent enough time on this I'll ack this.

Reply:
I would prefer that, yeah

Reply:
:+1: